### PR TITLE
[48404] Implemented message page iOS

### PR DESCRIPTION
**Pull request by jonasofrich**
IMG_0138
IMG_0137
IMG_0136

### CODE REVIEW

**Comment by pavshr** [comment]
Where is the label with last updated time?

**Comment by jonasofrich** [comment]
> Where is the label with last updated time?

Not sure what you mean. Can't find this in the sketches?

**Comment by pavshr** [comment]
> > Where is the label with last updated time?
> 
> Not sure what you mean. Can't find this in the sketches?

Yes, sorry. Looks like design templates are not updated. I will clarify it. For now this should not be merged, but review can be done.

**Review by pavshr — CHANGES_REQUESTED** [review]
Awaiting designer clarifications

**Comment by nc-kano** [comment]
@pavshr i think we can go with this view for now. we will correct this in the future as it is implemented in the same way on Android

**Comment by jonasofrich** [comment]
Added last updated label. New screenshots in top comment.

**Review by nc-kano — APPROVED** [review]

File: NDB.Covid19/NDB.Covid19/Locales/nn.json [hunk 1]
```diff
@@ -220,6 +220,8 @@
   "MESSAGES_LAST_UPDATED_LABEL": "Sist oppdatert {0} kl. {1}",
   "MESSAGES_RECOMMENDATIONS_": "Viktig – les kva du skal gjere",
   "MESSAGES_NOMESSAGES_HEADER": "Du har ingen meldingar",
+  "MESSAGES_NEW_MESSAGES_HEADER": "Du har ei ny melding",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nn.json:223]: I'll update these in the excel sheet tomorrow

File: NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs [hunk 2]
```diff
@@ -64,11 +65,22 @@ public override void ViewWillDisappear(bool animated)
...
-            StyleUtil.InitLabelWithSpacing(NoItemsLabel1, StyleUtil.FontType.FontBold, MESSAGES_NO_ITEMS_TITLE, 1.25, 32, 34);
+            StyleUtil.InitLabelWithSpacing(LabelLastUpdate, StyleUtil.FontType.FontRegular, LastUpdateString, 1.14, 15, 17, UITextAlignment.Left);
+            int unreadMessages = (await MessageUtils.GetAllUnreadMessages()).Count;
+            int messages = (await MessageUtils.GetMessages()).Count;
+            string headerText = MESSAGES_NO_ITEMS_TITLE;
+            if (unreadMessages > 0)
+            {
+                headerText = MESSAGES_NEW_MESSAGES_HEADER;
+            }
+            else if (messages > 0)
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:79]: What if messages is 0?

Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:79]: Line 74

Comment (pavshr) [reply NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:79]: aha, then it is default. yeah, I got it :D

**Review by pavshr — APPROVED** [review]

**Pull request merged**